.travis.yml

```diff
@@ -7,10 +7,7 @@ env:
   - NODE_VERSION="6.9"
 before_install:
   - nvm install $NODE_VERSION
-  - pip install codecov
-install:
-  - make deps
```

Comment:
I think it'd be nice to install all the dependencies as part of Travis's `install` stage rather than its `script` stage. But the Python dependencies actually need to be installed by tox in tox's venv, not by pip normally, so we'd need to do some fiddling to install tox and the JavaScript deps and then install the Python deps in tox but not actually run the tests yet. I'm sure it can be done but I decided not worth the complexity. Just run `make test` as the `script` and it'll install all necessary deps as part of the `script` stage